src/components/ViewRequest/NotesForm.tsx

```diff
@@ -0,0 +1,151 @@
+  Label,
+  Textarea,
+} from "@fluentui/react-components";
+import { useAddNote } from "api/notesApi";
+import { PeoplePicker } from "components/PeoplePicker/PeoplePicker";
+import { ContactIcon, TextFieldIcon } from "@fluentui/react-icons-mdl2";
+import { useState } from "react";
+import { Person, useRequest } from "api/requestsApi";
+import { useSendEmail } from "api/emailApi";
+
+declare const _spPageContextInfo: {
+  userId: number;
+  userDisplayName: string;
+  userEmail: string;
+  userLoginName: string;
+  webAbsoluteUrl: string;
+};
+
+const NotesForm = ({
+  isOpen,
+  closeDrawer,
+}: {
+  isOpen: boolean;
+  closeDrawer: () => void;
+}) => {
+  const params = useParams();
+  const requestId = Number(params.requestId);
+  const request = useRequest(requestId);
+  const addNote = useAddNote(requestId);
+  const sendMail = useSendEmail();
+
+  const [toSelections, setToSelections] = useState<Person[]>([]);
+  const [ccSelections, setCcSelections] = useState<Person[]>([]);
+  const [newNoteText, setNewNoteText] = useState("");
+
+  const subject =
+    "Action Required for RPA Tracker Request " +
+    requestId +
+    ", " +
+    request.data?.officeSymbol +
+    ", " +
+    request.data?.mpcn;
+
+  const body =
+    `A new note has been added by ${_spPageContextInfo.userDisplayName}<br><br>` +
+    `Link to request: <a href="${_spPageContextInfo.webAbsoluteUrl}/app/index.aspx#/Request/${requestId}">${_spPageContextInfo.webAbsoluteUrl}/app/index.aspx#/Request/${requestId}</a><br><br>` +
+    `Note contents:<br>${newNoteText}`;
+
+  if (
+    !isOpen &&
+    (toSelections.length > 0 || ccSelections.length > 0 || newNoteText)
+  ) {
+    setToSelections([]);
+    setCcSelections([]);
+    setNewNoteText("");
+  }
+
+  return (
+    <DrawerBody>
+      <Label id="To" weight="semibold" className="fieldLabel">
+        <ContactIcon className="fieldIcon" />
+        To
+      </Label>
+      <PeoplePicker
+        ariaLabel="To"
+        aria-labelledby="To"
+        selectedItems={toSelections}
+        itemLimit={10}
+        updatePeople={(items) => {
+          if (items?.[0]?.Title) {
+            setToSelections(items);
+          } else {
+            setToSelections([]);
+          }
+        }}
+      />
+      <br />
+
+      <Label id="Cc" weight="semibold" className="fieldLabel">
+        <ContactIcon className="fieldIcon" />
+        Cc
+      </Label>
+      <PeoplePicker
+        ariaLabel="Cc"
+        aria-labelledby="Cc"
+        selectedItems={ccSelections}
+        itemLimit={10}
+        updatePeople={(items) => {
+          if (items?.[0]?.Title) {
+            setCcSelections(items);
+          } else {
+            setCcSelections([]);
+          }
+        }}
+      />
+      <br />
+
+      <Label id="Body" weight="semibold" className="fieldLabel">
+        <TextFieldIcon className="fieldIcon" />
+        Body
+      </Label>
+      <Textarea
+        id="Body"
+        style={{ width: "100%", display: "block" }}
```

Comment:
Text area isn't playing nice with your styling here.  It makes the box appear 100% but actual text input area is only the left third.